### ISSUE TITLE
Public Access: login button not available after logout from meeting

### ISSUE THREAD

**Issue by MSoeb** [OpenSlides/openslides-client#4417]
**Describe the bug**
If you log out of a meeting, the button for public access is missing in the login screen. This is because the URL still contains the meeting. 

**How to Reproduce**
1. Activate in orga settings the public access
2. open a meeting
3. logout from the meeting
4. -> button is missing

**Expected behavior**
When logging out of a meeting, you should be redirected to the instance url. 

If this is not an option or leads to problems, a login button for public access should be available on the meeting subpages.

**Comment by MSoeb** [comment]
The issue has a connection to #4281

**Comment by reiterl** [comment]
I can reproduce this with a meeting which is public and another non-public meeting, from which I logout.
We had the discussion about the login URL after the logout of a meeting. This is another side effect, which should not be relevant in production. Won't fix? @rrenkert please look at this.

**Comment by MSoeb** [comment]
FYI @reiterl @rrenkert : Did I get it right, that this should not happen in production instances? If yes, this isn't the case. I've tried it on a live instance and the button is still missing after logout from a meeting. I think in general the missing redirection to the instance URL after logout is the problem.

**Comment by rrenkert** [comment]
@reiterl Can you please have a look? as @MSoeb mentioned this dows not only happen in dev mode but also in production.

**Comment by reiterl** [comment]
Yes, this happens also in prod. @rrenkert have you read *my* comment about it?

**Comment by bastianjoel** [comment]
There was another problem described in #4541 which fix will also fix this issue.
